src/styles/Colors.scss

```diff
@@ -37,3 +37,7 @@ $LIGHT-GREY:  #BBBBBB;
 
 //Long Tag color
 $LIGHT-GRAY:#F5F5F5;
+
+//Error color
+$ERROR: #F63E7C;
```

Comment:
Could you please confirm if this color code is correct? as per DDS, I see both icon and text are #CE1126

Reply:
Changed it to CE1126.